src/frontend-pwa/src/views/Report/Report.tsx

```diff
@@ -1,12 +1,152 @@
-import { TextField } from '@mui/material';
+/* eslint-disable react/no-array-index-key */
+/**
+ *
+ * @summary - Event reporting page for the Wayfinder application.
+ *          - Users can submit reports about events they witness.
+ *
+ * @author  TylerMaloney
+ */
+import React, { useState, useEffect, useCallback } from 'react';
+import axios from 'axios';
+import { Button } from '../../components/Button/Button';
+import {
+  StyledReportContainer,
+  StyledReportOuterDiv,
+  StyledTextArea,
+  Section,
+  StyledSelect,
+  StyledInput,
+  StyledP,
+} from './report.styles';
+import constants from '../../constants/Constants';
+import useAppService from '../../services/app/useAppService';
 
 export default function Report() {
+  const [eventType, setEventType] = useState('');
+  const [details, setDetails] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isFormValid, setIsFormValid] = useState(false);
+  const { state } = useAppService();
+  const latitude = state.currentLocation ? state.currentLocation.lat : 49.2827;
+  const longitude = state.currentLocation ? state.currentLocation.long : -123.2;
+  const eventOptions: Array<string> = [
+    'Damaged Infrastructure',
+    'Animal Sighting',
+    'Suggestion/Complaint',
+    'Miscellaneous',
+  ];
+
+  const checkFormValidity = useCallback(() => {
+    const isEventTypeValid = !!eventType;
+    const isDetailsValid = details.length >= 10 && details.length <= 256;
```

Comment:
If you use the search bar as a reference you'll remember we styled a div to make the search bar and buttons all blend together to create one harmonious looking container
We can apply that same practice to the textarea in the Report module by adding a character count to the bottom right
This can give the user meaningful information about how many characters they're allowed, as they type, with less onBlur error messages being important as well. 

This is a sort of tutorial in creating one in Javascript that you can borrow from 
https://levelup.gitconnected.com/character-counter-for-text-area-element-using-javascript-72d9a7ff5f77